### DISCUSSION TITLE
Error: d.secretNames.map is not a function

### DISCUSSION THREAD

**Discussion by BassemMagdi** [AntlerVC/firetable#1544] [Support / Q&A]
👉 **Please describe the steps that you took that led to this bug.**

**Comment by alorenzo91** [comment]
I am experiencing the same issue, but after reviewing further it looks like the rowy service account requires the IAM policy secretmanager.secrets.list when provided this error is resolved.

**Reply by alorenzo91** [reply]
To give more context in my issue. This error would appear when attempting to configure an action column.
![image](https://github.com/user-attachments/assets/7b9c8bf9-8f8c-416f-b400-6cebe047c90e)
